magento/devdocs#: Develop data and schema patches. Revert module datapatch (#4504)
magento/devdocs: guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md

    }
```

## Reverting data patches

> Magento does not allow the reverting of a particular module data patch. It only allows the reverting of all module data patches per the `module:uninstall` command.

Reviewer: ```suggestion
Magento does not allow you to revert a particular module data patch. However, you can revert all `composer` installed or `non-composer` installed data patches using the `module:uninstall` command.
```

Author: **fixed**

> Run the folowing command to revert all the module's data patches (only for modules that have been installed through `composer`):

Reviewer: ```suggestion
Run the following command to revert all `composer` installed data patches:
```

Author: **fixed**

```bash
bin/magento module:uninstall Vendor_ModuleName
```

> To revert all `non composer` data patches, run:

Reviewer: ```suggestion
Run the following command to revert all `non-composer` installed data patches:
```

Author: **fixed**

Author: @erikmarr , request changes have been implemented. Could you please review?

Thank you!
